### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by kuttattz**
# Ansible modules for Unity release version 2.0.0 includes:
Added support for Unity Puffin v5.4

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| #47  |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit Test
- [x] Functional Test
- [x] Regression Test

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,14 @@ Dellemc.Unity Change Log
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
+-------------
+
+- Adding support for Unity Puffin.
```
Comment (trisha-dell) [inline CHANGELOG.rst:14]: Mention version number as well.

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -174,3 +174,8 @@ releases:
       minor_changes:
       - Patch update to fix import errors in utils file.
     release_date: '2023-07-31'
+  2.0.0:
+    changes:
+      major_changes:
+      - Adding support for Unity Puffin.
+    release_date: '2024-03-31'
```
Comment (trisha-dell) [inline changelogs/changelog.yaml:181]: 2024-03-29

File: docs/Release Notes.md [hunk 3]
```diff
@@ -38,9 +38,7 @@ New features & enhancements
 ---------------------------
 This release has the following changes -
 
-- Support addition of host from the Host List to NFS Export in nfs module.
-- Support enable/disable advanced dedup in volume module.
-- Add synchronous replication support for filesystem.
+- Adding support for Unity Puffin.
```
Comment (trisha-dell) [inline docs/Release Notes.md:41]: add version number as well.

File: meta/runtime.yml [hunk 4]
```diff
@@ -1,79 +1,2 @@
 ---
-requires_ansible: ">=2.13"
```
Comment (sachin-apa) [inline meta/runtime.yml:2]: We should not remove this , instead we should tombstone it.. 
Please see the details [here ](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#tombstoning-a-module-or-plugin-in-a-collection)

File: CHANGELOG.rst (hunk 1 above)
Comment (kuttattz) [reply CHANGELOG.rst]: Done

File: changelogs/changelog.yaml (hunk 2 above)
Comment (kuttattz) [reply changelogs/changelog.yaml]: Done

File: docs/Release Notes.md (hunk 3 above)
Comment (kuttattz) [reply docs/Release Notes.md]: Done

File: meta/runtime.yml (hunk 4 above)
Comment (kuttattz) [reply meta/runtime.yml:2]: Done

**Review by sachin-apa — COMMENTED** [review]
1.  Please move requirements.txt from _tests_ folder to _tests/unit_  for unit tests to install dependency. 
2.  Add a codecov.yml file for the path correction of coverage reports.

@Jennifer-John  Can you please check, looks like GitHub Actions are not ran here.

**Comment by Jennifer-John** [comment]
> 1. Please move requirements.txt from _tests_ folder to _tests/unit_  for unit tests to install dependency.
> 2. Add a codecov.yml file for the path correction of coverage reports.
> 
> @Jennifer-John Can you please check, looks like GitHub Actions are not ran here.

Enabled the workflow , @kuttattz could you push /squash the commits, we can check the on triggered workflow

**Review by Jennifer-John — DISMISSED** [review]

**Review by trisha-dell — APPROVED** [review]

**Review by felixs88 — APPROVED** [review]

**Review by Jennifer-John — APPROVED** [review]

**Pull request merged**